Feed/Feed.php

```diff
@@ -104,9 +104,29 @@ public function set($parameter, $value)
      */
     public function get($parameter, $default = null)
     {
+		if('link' == $parameter){
```

Comment:
In order to follow the Symfony code conventions, can you please update the line to add spaces after `if` statement and before opening bracket?

```php
if ('link' == $parameter) {
```